common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -48,6 +48,11 @@ export const bufferToString = (blob: ArrayBufferLike, encoding: string): string
  * Minimal implementation of Buffer for our usages in the browser environment.
  */
 export class IsoBuffer extends Uint8Array {
+    // Need to have ctor for it to be in proto chain for instanceof check in from() method to work
+    public constructor(buffer: ArrayBufferLike, byteOffset?: number, length?: number) {
```

Comment:
I do not know if ctor is required. When I tested it in debugger without this change, prototype chain did not include IsoBuffer and my instanceof test failed.
However here, when I run jest tests, this seems to be not required. Not sure...

Happy to be educated here on what I'm missing. But if there is no good explanation, I'll keep it here.